src/sqlast/query.rs

```diff
@@ -177,9 +177,9 @@ impl ToString for Cte {
 #[derive(Debug, Clone, PartialEq, Hash)]
 pub enum SQLSelectItem {
     /// Any expression, not followed by `[ AS ] alias`
-    UnnamedExpression(ASTNode),
+    UnnamedExpression(Expr),
```

Comment:
Should we keep `UnnamedExpression` and `ExpressionWithAlias` or rename them to use `Expr`? Same for parse_.._expression() methods.

Reply:
Oh, good idea! Done.